udata/migrations/2016-06-24-rename-territories.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Update spatial granularities with French names.
+ *
+ */
+
+var datasets = db.dataset.find({
+    'spatial.granularity': 'fr/town'
+});
+
+print('Candidates: ' + datasets.count());
+
+var counter = 0;
+
+datasets.forEach(function (dataset) {
+    var result = db.dataset.update({_id: dataset._id},
+                                   {$set: {'spatial.granularity': 'fr/commune'}});
+    counter += result.nModified;
+});
+print(counter, 'datasets granularity modified for towns.');
+
+datasets = db.dataset.find({
+    'spatial.granularity': 'fr/commune'
+});
+
+counter = 0;
+
+datasets.forEach(function (dataset) {
+    var oldZones = dataset.spatial.zones;
+    if (!oldZones) return
+    var newZones = oldZones.map(function (zones) {
+        if (typeof zones === 'string') {
```

Comment:
Zones should always be a list, it shouldn't happen.
If it's the case, may be perform another migration before to fix that